server/src/test/java/org/spine3/server/stand/StandFunnelShould.java

```diff
@@ -53,6 +70,56 @@ public void initialize_properly_with_stand_only() {
         Assert.assertNotNull(standFunnel);
     }
 
+    @Test
+    public void initialize_properly_with_various_builder_options() {
+        final Stand stand = TestStandFactory.create();
+        final Executor executor = Executors.newSingleThreadExecutor(new ThreadFactory() {
+            @Override
+            public Thread newThread(Runnable r) {
+                return Thread.currentThread();
+            }
+        });
+
+        final StandFunnel blockingFunnel = StandFunnel.newBuilder()
+                                                      .setStand(stand)
+                                                      .setExecutor(executor)
+                                                      .build();
+        Assert.assertNotNull(blockingFunnel);
+
+        final StandFunnel funnelForBusyStand = StandFunnel.newBuilder()
+                                                          .setStand(stand)
+                                                          .setExecutor(Executors.newSingleThreadExecutor())
+                                                          .build();
+        Assert.assertNotNull(funnelForBusyStand);
+
+
+        final StandFunnel emptyExecutorFunnel = StandFunnel.newBuilder()
+                                                           .setStand(TestStandFactory.create())
+                                                           .setExecutor(new Executor() {
```

Comment:
Also, I don't see a difference between initialisation with different executors. These cases will differ in an operation behaviour, not in the initialisation.